cmd/rook/ceph/cleanup.go

```diff
@@ -44,6 +41,9 @@ func init() {
 	cleanUpCmd.Flags().StringVar(&dataDirHostPath, "data-dir-host-path", "", "dataDirHostPath on the node")
 	cleanUpCmd.Flags().StringVar(&namespaceDir, "namespace-dir", "", "dataDirHostPath on the node")
 	cleanUpCmd.Flags().StringVar(&monSecret, "mon-secret", "", "monitor secret from the keyring")
+	cleanUpCmd.Flags().StringVar(&clusterFSID, "cluster-fsid", "", "ceph cluster fsid")
+	cleanUpCmd.Flags().StringVar(&clusterName, "cluster-name", "", "ceph cluster name")
+	cleanUpCmd.Flags().StringVar(&sanitizeDiskConfirmation, "sanitize-disk-confirmation", "", "whether to sanitize the disk(s)")
```

Comment:
We should only need the confirmation to decide launching the job, right? No need to pass the flag into the job.

Reply:
Yes leftover my bad!